Moli/Core/Screens/Profile/ProfileImageAndBannerView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Mantis
+
+enum ImageType {
+    case profile, banner
+}
+
+struct ProfileImageAndBannerView: View {
```

Comment:
:art: Det här är nog min name convention OCD, men det kanske ska istället stå `ProfileAndBannerImageView` eftersom det är både Profile och banner images som ingår här? 😛 
```suggestion
struct ProfileAndBannerImageView: View {
```